src/commcare_cloud/ansible/setup_pg_logical_replication.yml

```diff
@@ -144,7 +144,7 @@
       postgres
       postgresql_query:
         db: '{{ item.target_db_name }}'
-        query: "GRANT USAGE ON SCHEMA pglogical TO {{ secrets.POSTGRES_USERS.commcare }};"
+        query: "GRANT USAGE ON SCHEMA pglogical TO {{ postgres_users.commcare }};"
```

Comment:
@dannyroberts I assume this should also be applied to other places e.g. inside env config files for ICDS

Reply:
Yup. I didn't make a big stink about it because for environments using the ansible-vault backend, `secrets.*` syntax should continue to work as before, so if that's strictly in config files, I'd maybe encourage switching syntax but it's up to you